modules/backup-and-restore/pages/backup-cluster.adoc

```diff
@@ -39,6 +39,11 @@ $ gadmin backup create weekly
 
 This creates a backup file named `weekly-<timestamp>` at the xref:configurations.adoc[backup location you configured].
 
+You can also make `weekly` a complete tag by using the flag `--custom-tag`:
+----
+$ gadmin backup create weekly --custom-tag
+----
+This creates a backup file named `weekly` without the `-<timestamp>` suffix.
```

Comment:
nit: `backup file` makes me feel the backup will be a single file, but it is a directory name on multiple nodes. how about removing the word "file"?  i.e. "backup file" -> "backup" and line 40 has the same problem.

Reply:
good point!

Reply:
applied